Version/VersionInfo.cs

```diff
@@ -14,6 +15,11 @@ public static string GetEmuVersion()
 		return DeveloperBuild ? ("GIT " + SubWCRev.GIT_BRANCH + "#" + SubWCRev.GIT_SHORTHASH) : ("Version " + Mainversion);
 	}
 
+    public static string GetEmuVersionNonDev()

```

Comment:
Mainversion is public, why not obtain it directly?

Reply:
done